pkg/printers/internalversion/describe.go

```diff
@@ -583,7 +589,14 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 	})
 }
 
-func printControllers(annotation map[string]string) string {
+func printController(controllee metav1.Object) string {
+	if controllerRef := controller.GetControllerOf(controllee); controllerRef != nil {
+		return fmt.Sprintf("%s/%s", controllerRef.Kind, controllerRef.Name)
+	}
+	return ""
+}
+
+func printCreator(annotation map[string]string) string {
```

Comment:
I am wondering if created-by is used for anything else that the controller ref cannot. Can you spawn a separate issue to deprecate it and start removing it from places we can while keeping backwards compatibility?